src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,21 @@
+package nextstep.ladder.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Ladder {
+
+    private List<Line> ladder;
+
+    public Ladder(int ladderHeight, int userCount) {
+        RandomMakeStrategy randomMakeStrategy = new RandomMakeStrategy();
+        ladder = Stream.generate(() -> new Line(userCount, randomMakeStrategy)).limit(ladderHeight).collect(Collectors.toList());
```

Comment:
```suggestion
        ladder = Stream.generate(() -> new Line(userCount, randomMakeStrategy))
        .limit(ladderHeight)
        .collect(Collectors.toList());
```

스트림은 행위단위로 내려쓰는것이 가독성과 디버깅에 장점이 많다고 생각해요.